docs/source/config_file.rst

```diff
@@ -197,18 +197,19 @@ section of the command line docs.
 
 .. confval:: exclude
 
-    :type: newline separated list of regular expressions
+    :type: regular expressions
 
-    A newline list of regular expression that matches file names, directory names and paths
+    A regular expression that matches file names, directory names and paths
     which mypy should ignore while recursively discovering files to check.
     Use forward slashes on all platforms.
 
     .. code-block:: ini
 
       [mypy]
-      exclude =
+      exclude = (?x)(
```

Comment:
It might be worth it to note that the `x` flag causes whitespace, and lines beginning with `#`, to be ignored inside the pattern. A link to <https://docs.python.org/3/library/re.html#re.X> would save having to include further explanations.

Reply:
Something like:

`(?x)` can be used as the first part of a regular expression. This corresponds to setting the [`re.X`](https://docs.python.org/3/library/re.html#re.X) flag, which causes most witespace in the pattern to be ignored. Everything after an unescaped `#` is also ignored, allowing for comments. [The Python `re` module has more information.](https://docs.python.org/3/library/re.html#re.X)

Reply:
I tinkered with it a bit. Let me know what you think.

<img width="710" alt="Screen Shot 2022-01-01 at 19 12 39" src="https://user-images.githubusercontent.com/222581/147863591-5bee7e5c-d504-4395-b6ae-3347a252f8a5.png">

The hyperlink goes to https://docs.python.org/3/library/re.html#re.X.

Reply:
I love it! Thanks for accepting feedback.